<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-18T11:51:18Z" name="Default suite" finished-at="2023-09-18T11:51:26Z" duration-ms="8233">
    <groups>
    </groups>
    <test started-at="2023-09-18T11:51:18Z" name="Default test" finished-at="2023-09-18T11:51:26Z" duration-ms="8233">
      <class name="search.TestCase">
        <test-method signature="openUrl()[pri:0, instance:search.TestCase@6c2c1385]" started-at="2023-09-18T11:51:18Z" name="openUrl" finished-at="2023-09-18T11:51:22Z" duration-ms="3631" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openUrl -->
        <test-method signature="serachCentex()[pri:0, instance:search.TestCase@6c2c1385]" started-at="2023-09-18T11:51:22Z" name="serachCentex" finished-at="2023-09-18T11:51:24Z" duration-ms="2230" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- serachCentex -->
        <test-method signature="validateTradingSymbol()[pri:0, instance:search.TestCase@6c2c1385]" started-at="2023-09-18T11:51:24Z" name="validateTradingSymbol" finished-at="2023-09-18T11:51:26Z" duration-ms="2329" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[tradingSymbol is not equal to CENTEX but it is :CENTEXT-BE expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: tradingSymbol is not equal to CENTEX but it is :CENTEXT-BE expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at search.TestCase.validateTradingSymbol(TestCase.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTradingSymbol -->
      </class> <!-- search.TestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
